src/Components/Publishing/__test__/ArticleTracking.test.tsx

```diff
@@ -1,10 +1,16 @@
-import { mount } from "enzyme"
 import "jest-styled-components"
-import React from "react"
-import Events from "../../../Utils/Events"
-import { Article } from "../Article"
+
+import { mount } from "enzyme"
+import Enzyme from "enzyme"
+import Adapter from "enzyme-adapter-react-16"
+import * as React from "react"
+
+import Events from "../../../utils/events"
+import { Article } from "../article"
 import { getArticleFullHref } from "../Constants"
-import { StandardArticle } from "../Fixtures/Articles"
+import { StandardArticle } from "../fixtures/articles"
+
+Enzyme.configure({ adapter: new Adapter() })
```

Comment:
This should be configured in our Jest setup script, which gets loaded before each suite runs:  https://github.com/artsy/reaction/blob/master/src/setup_jest.js

Reply:
Ah! just scrolled down 😄  maybe just remove? Why was this needed here?

Reply:
Yeah lol sorry, we don't need it here, it's already in the setup script